web/src/components/GameDashboard.jsx

```diff
@@ -57,16 +57,15 @@ export default function GameDashboard({ loading, games, deleteGame, cloneGame, g
     if (games.length >= 1) {
       return games
         .map((game, index) => {
-          const { id, title, description, cluster, domain, grade, standard } = game;
+          const { id, title, description, cluster, domain, grade, standard, imageUrl } = game;
           const questionCount = game?.questions?.length || 0;
-          //const image = getGameImage(game); // figure out what this function does or should do
-          const image = null;
+          
           return (
             <Grid container item xs={12} md={addquestion ? 12 : 6} lg={addquestion ? 12 : 4}>
-              <Card className={classnames(classes.game, !match && classes.gameGrid, match && Number(match.params.gameIndex) === index + 1 && classes.gameSelected)} key={id} onClick={() => onClickGame(game.id, gamemakerIndex)}>
+              <Card className={classnames(classes.game, !match && classes.gameGrid, match && Number(match.params.gameIndex) === index + 1 && classes.gameSelected)} key={id} onClick={() => onClickGame(game.id, gameId)}>
```

Comment:
Also curious if this would need to be Number(match.params.gameId) instead of gameIndex ?